config/crd/patches/additional_supported_versions_patch_licensemanagers.yaml

```diff
@@ -0,0 +1,78 @@
+- op: add
```

Comment:
remove this we do not need patch for older version as this will be new CR

Reply:
Done